internal/pkg/util/env/create.go

```diff
@@ -95,15 +95,29 @@ func overridesForContainerEnv(g *generate.Generator, hostEnvs []string) envKeyMa
 
 // warning if deprecated keys are set
 func warnDeprecatedEnvUsage(hostEnvs []string) {
+	envMap := make(map[string]string)
+	for _, env := range hostEnvs {
+		strs := strings.SplitN(env, "=", 2)
+		if len(strs) == 2 {
+			envMap[strs[0]+"="] = strs[1]
```

Comment:
This overall looks pretty good to me, except I don't see the added value of including the `+"="` here and below in the lookup.

Reply:
thanks, it also looks like that we do not need the "=" as the part of the key